src/widgets/requirements/ui/requirements.tsx

```diff
@@ -1,43 +1,35 @@
-import { LINKS } from '@/app/const';
+import classNames from 'classnames/bind';
 import { LinkCustom } from '@/shared/ui/link-custom';
 import { List } from '@/shared/ui/list';
 
-import './requirements.scss';
+import { buttonLink, buttonVariant, textContent } from '@/widgets/requirements/constants.ts';
 
-const mentorRequirements = [
-  "Desire to help students. If you've been working with JS/TS in production for more than 6 months, then that's great",
-  'Desire to mentor 2 to 6 students online or in person',
-  'Ability to spend 3 to 5 hours per week',
-];
+import styles from './requirements.module.scss';
 
-const mentorResponsibilities = [
-  'Conducting an interview',
-  'Code review tasks',
-  "Answers to students' questions",
-];
+const cx = classNames.bind(styles);
 
 export const Requirements = () => {
   return (
-    <section className="requirements container">
-      <div className="requirements content">
-        <div className="requirements-info">
-          <div className="requirements-list">
-            <div className="title">Requirements for mentors</div>
-            <List data={mentorRequirements} />
+    <section className={cx('container')}>
+      <article className={cx('content')}>
+        <div className={cx('requirements-info')}>
+          <div className={cx('requirements-list')}>
```

Comment:
```suggestion
          <div className={cx('requirements-list-wrapper')}>
```
Maybe it's worth changing the class name a little here, for example `requirements-list-wrapper`, since as far as I can see the list is below

Reply:
Done